tutorials/atomicallyResolveReconstruction/tutorial_atomicallyResolveReconstruction.m

```diff
@@ -110,7 +119,7 @@
 % The variable |isChemaxonInstalled| contains a logical value defined by the user 
 % if the license is installed or not.
 
-isChemaxonInstalled = false; % Change varibale to "true" if ChemAxon is installed
+isChemaxonInstalled = true; % Change varibale to "true" if ChemAxon is installed
```

Comment:
@fplanes, may you change this to `false`? Otherwise, the tutorial will fail on the CI server. Thanks 👍